frontend/src/component/Sidebar/SidebarItem.jsx

```diff
@@ -6,6 +6,11 @@ import { isMobile } from 'react-device-detect'
 import { Icon } from 'tracim_frontend_lib'
 
 const SidebarItem = (props) => {
+  const handleClickOnSidebarItemButton = () => {
+    if (isMobile && !props.isSidebarClosed) props.onClickToggleSidebar()
```

Comment:
I'm not a fan of passing a handler through many child components to manage an edge case.
Could you, instead, use event propagation and put this intelligence on a higher component like Sidebar.jsx?

Reply:
The problem is in that case we have to "block" the toggle for some elements. On https://github.com/tracim/tracim/pull/5870/commits/8126dea6d7edeecf8c2a1355dc9e0e7cd0f90c0d I made the version putting the intelligence on Sidebar. What do you think?

Reply:
as seen together:
- have only one handler on top (component hierarchy) of the sidebar. It will handle alone any click on the sidebar whether it has to open or close the sidebar or not
- remove any other onClick handlers related to opnening and closing the sidebar (also remove them from proptypes)
- create a class named somehting like "lockSidebarOpenCloseAnimation"
- merge the intelligence of handleClickToggleSidebar and handleClickSidebarWhenOpenOnMobile
- put the class lockSidebarOpenCloseAnimation on every DOM element we don't want them to open/close the sidebar

Reply:
fixed